src/test/kotlin/org/rustSlowTests/cargo/runconfig/buildtool/CargoBuildManagerTest.kt

```diff
@@ -903,6 +903,12 @@ class CargoBuildManagerTest : RunConfigurationTestBase() {
 
     @Suppress("EqualsOrHashCode")
     companion object {
+        private const val ENABLED: Boolean = false
+
+        private fun doTest(action: () -> Unit) {
```

Comment:
Maybe overriding `com.intellij.testFramework.UsefulTestCase#shouldRunTest` is better?

Reply:
Fixed.